pkgs/data/icons/capitaine-cursors/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, stdenv, fetchFromGitHub
+, srcRepo ? false, inkscape ? null, xcursorgen ? null}:
```

Comment:
I think this should always be true.

Reply:
Oh alright, just put that option since there was a prebuilt theme in `dist/`